projects/cashmere/src/lib/datepicker/datepicker.component.scss

```diff
@@ -0,0 +1,296 @@
+@import '../sass/colors';
+@import '../sass/typography';
+@import '../sass/functions';
+@import '../sass/mixins';
+
+.hc-datepicker-content {
+    width: 296px;
+    height: 354px;
```

Comment:
I think we could use a little extra padding on the bottom of the container too - maybe height: 360px

Reply:
I ended up removing the height and width and let the elements size themselves and things "fit" much better

Reply:
nevermind, the month and year views need the height and width :P

Reply:
Datepicker container is looking much better!  Now that we got rid of the month label though, it's a bit long.  For `.hc-datepicker-content`, 330px looked about right to me.